assets/components/src/plugin-installer/index.js

```diff
@@ -187,15 +203,15 @@ class PluginInstaller extends Component {
 								notificationLevel="error"
 							/>
 						);
+						<Button
```

Comment:
The `Button` is after the `return`, so it will never get rendered? It never showed up in my testing.

Reply:
This is correct actually. The `<Button>` is only needed in the normal non-`asProgressBar` context, since `asProgressBar` causes the plugins to begin installing as soon as the Component mounts. It was confusing that I made this change in the same PR—ideally I would have done this one separately.

Reply:
I mean it will never render in any context (it's within the `map` after the `return`):
```
slugs.map( slug => {
    return (
	    <ActionCard/>
    );
    <Button/>
})
```
You can see no button on e.g. the Subscriptions Wizard:
![localhost_wp-admin_admin php_page=newspack-subscriptions-wizard (1)](https://user-images.githubusercontent.com/7317227/60286282-4333f000-98c4-11e9-8e59-81cce27eba4f.png)

Reply:
My mistake, you are correct. This should fix: https://github.com/Automattic/newspack-plugin/pull/132/commits/014bc4313c4b0b163c57314901ebe36c909e3611